mscore/propertymenu.cpp

```diff
@@ -588,9 +588,10 @@ void Ms::ScoreView::selectInstrument(InstrumentChange* ic)
                   Instrument instr = Instrument::fromTemplate(it);
                   ic->setInit(true);
                   ic->setStaffGroup(it->staffGroup);
-                  ic->clearLines();
                   if (it->staffTypePreset)
                         ic->setLines(it->staffTypePreset->lines());
+                  else
+                        ic->setLines(5);
```

Comment:
why the magic `5` here, and the removal of `clearLines`?

Reply:
When lines was a vector, when I was storing the number of lines for each stave, it would clear it before setting up the instrument to make sure it was a clean setup, but as it's now just storing the default number of lines, it's unnecessary. And standard instruments don't have a stafTypePreset, for some reason, so this was to set the number of lines to 5 manually. I've set this as a default value in the header file, which should fix the tests, although its still necessary to keep it here to facilitate changing from a percussion/tab instrument to a standard one.

Reply:
Huh, that's a bit odd. Should be fine though.